src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/toolkit/util/CommentHelper.java

```diff
@@ -168,11 +156,6 @@ public List<? extends DocTree> getDescription(DocTree dtree) {
         return getTags(dtree);
     }
 
-    public TypeElement getReferencedClass(DocTree dtree) {
-        Element e = getReferencedElement(dtree);
-        return getReferencedClass(e);
-    }
-
     public TypeElement getReferencedClass(Element e) {
```

Comment:
For what it's worth, I dislike the remaining `getReferenced`_Thing_ methods remaining in this class, since they are just null-safe wrappers around methods in `Utils` that are just wrappers for code in `java.lang.model`. But that's another more-manual cleanup for another day.